mugoh/qstioner
# #163841854 A User will Scroll Infinitely when Viewing Meetups

#### What does this PR do?
Allow users to scroll with no stop when viewing meetups 

#### Description of Task to be completed?
- Append meetup elements to page when user scrolls
- Show login modal for new users during a prefixed scroll height

#### How should this be manually tested?
Ensure the server is running. Follow the steps given [here](https://github.com/hogum/qstioner-v2) to do so.
1. Clone the repo in your machine
``` shell
$ git clone https://github.com/hogum/qstioner
```
2. Navigate to the directory `qstioner` where you have the repo cloned
``` shell
$ cd qstioner
```
3. Checkout this Feature branch
``` shell
$ git checkout ft-infinite-scroll-163841854
```
4. Open the `index.html` file
``` shell
$ browser-name index.html
```
or
Simply  load up on your preferred browser
5. Navigate to the sign in page and sign in with account credentials.
- email: `admincow@mammals.milk` password: `pa555word`

6. Create a meetup on the admin dashboard.

7. Open  this meetup by clicking on it's title. A page opens where questions can be added to the meetup.
 Created meetups should be visible on the meetups display area.

8. Scroll to the bottom of the meetup display container. The scroll effect should be continous.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#163841854](https://www.pivotaltracker.com/story/show/163841854)
